doc/manual/installation.md

```diff
@@ -50,7 +50,7 @@ Import node.js repository (can be skipped on Ubuntu and Debian Jessie):
 
 Install the required packages (needed to compile Ruby and native extensions to Ruby gems):
 
-    sudo apt-get install -y runit build-essential git zlib1g-dev libyaml-dev libssl-dev libgdbm-dev libreadline-dev libncurses5-dev libffi-dev curl openssh-server checkinstall libxml2-dev libxslt-dev libcurl4-openssl-dev libicu-dev logrotate python-docutils pkg-config cmake nodejs graphviz
+    sudo apt-get install -y runit build-essential git zlib1g-dev libyaml-dev libssl-dev libgdbm-dev libreadline-dev libncurses5-dev libffi-dev curl openssh-server checkinstall libxml2-dev libxslt-dev libcurl4-openssl-dev libicu-dev logrotate python-docutils pkg-config cmake nodejs graphviz rake
```

Comment:
@rvictory That makes sense, I wonder why that never caused any problems in my tests. Can you move it to the ruby section where we install the `bundler` and `forman` gem? I am afraid the `rake` package might depend on an old ruby version in the package manager.